### PR TITLE
Automated leaderboard reporting

### PR DESCRIPTION
**Pull request by Unimatrix0**

### CODE REVIEW

**Comment by dashed** [comment]
rebase; and make this opt-in.

**Comment by Unimatrix0** [comment]
I'ts already opt-in, no?

File: robin.user.js [hunk 1]
```diff
@@ -1,9 +1,9 @@
 // ==UserScript==
 // @name         parrot (color multichat for robin!)
 // @namespace    http://tampermonkey.net/
-// @version      2.67
+// @version      2.7
```
Comment (jhon) [inline robin.user.js]: This should probably be 2.70

File: robin.user.js [hunk 2]
```diff
@@ -322,6 +322,9 @@
     Settings.addBool("tabChanColors", "Use color on regular channel messages in tabs", true);
     Settings.addBool("twitchEmotes", "<a href='https://twitchemotes.com/filters/global' target='_blank'>Twitch emotes</a>", false);
     Settings.addBool("timeoutEnabled", "Reload page after inactivity timeout", true);
+	Settings.addBool("reportStats", "Contribute statistics to the <a href='https://monstrouspeace.com/robintracker/'>Automated Leaderboard</a>.", false);
+    Settings.addInput("statReportingInterval", "Report Statistics Interval (seconds)", "600");
```
Comment (jhon) [inline robin.user.js]: As the Leaderboard guy, I'd prefer to see a 60s default interval.

Comment (Unimatrix0) [inline robin.user.js]: It's set to 600 because of the number of people who run the script. Do you really want everyone sending updates every minute?

Comment (jhon) [inline robin.user.js]: While the server could probably handle it, I understand your point. How about we compromise on 300s/5 min?

**Comment by jhon** [comment]
Opt-in and 5min default update interval. Looks good from my perspective.

**Comment by dashed** [comment]
@jhon is everything resolve from your perspective?

**Comment by jhon** [comment]
@dashed Yes.

**Comment by 5a1t** [comment]
OK for me as long as this is opt-in.

**Pull request merged**